facades/PC/build.gradle.kts

```
@@ -111,16 +111,10 @@ configurations {
     }
 }
 
-tasks.register<JavaExec>("game") {
-    description = "Run 'Terasology' to play the game as a standard PC application"
+// Used for all game configs.
+val commonConfigure : JavaExec.()-> Unit = {
```

Use function-definition syntax here instead of assigning an anonymous function to this variable?

```suggestion
fun JavaExec.commonConfigure(): Unit {
```

I _think_ that's what you were going for. I'm a little rusty.